exercise.js

```diff
@@ -0,0 +1,56 @@
+const addPerson = (request_obj, res) => {
+  if (!request_obj.files) {
+      return res.status(400).send("No files were uploaded.");
+  }
+
+  let Message = '';
+  let first_name = request_obj.body.first_name;
+  let last_name = request_obj.body.last_name;
+  let position = request_obj.body.position;
+  let int_number = request_obj.body.number;
+  let username = request_obj.body.username;
+  let UploadedFile = request_obj.files.image;
+  let image_name = UploadedFile.name;
+  let fileExtension = UploadedFile.mimetype.split('/')[1];
+
+  let usernameQuery = "SELECT * FROM `people` WHERE user_name = '" + username + "'";
+
+  db.query(usernameQuery, (err, result) => {
+      if (err) {
+          // Error
+          res.status(500).send(err);
+      }
+      if (result.length > 0) {
+          Message = 'Username already exists';
+          res.render('add-person.ejs', {
+              Message,
+              title: "Welcome to Socka | Add a new player"
+          });
```

Comment:
Message = 'Username already exists';
          res.render('add-person.ejs', {
              Message,
              title: "Welcome to Socka | Add a new player"
          }); 

handle in a function